routes/middleware/get_places.js

```diff
@@ -1,36 +1,27 @@
-const request = require('request');
+const axios = require('axios');
 
-module.exports = (req, res, next) => {
-  const url = 'https://nazareth-open-tourism-platform.herokuapp.com/api/v1/places';
+module.exports = async (req, res, next) => {
+  const url = `${process.env.URI}/places`;
+  const placesResponse = await axios.get(url);
+  res.locals.places = placesResponse.data
+    .map(place => {
```

Comment:
I would say maybe this map/filter should be moved to a helper function so you can test it separately.